service/controller/legacy/v30/adapter/guest_security_groups_test.go

```diff
@@ -188,14 +188,16 @@ func TestAdapterSecurityGroupsRegularFields(t *testing.T) {
 func TestAdapterSecurityGroupsKubernetesAPIRules(t *testing.T) {
 	t.Parallel()
 	testCases := []struct {
-		description            string
-		customObject           v1alpha1.AWSConfig
-		apiWhitelistingEnabled bool
-		apiWhitelistSubnets    string
-		elasticIPs             []*ec2.Address
-		hostClusterCIDR        string
-		expectedError          bool
-		expectedRules          []securityGroupRule
+		description                   string
+		customObject                  v1alpha1.AWSConfig
+		apiWhitelistingEnabled        bool
+		apiWhitelistSubnets           string
+		elasticIPs                    []*ec2.Address
+		hostClusterCIDR               string
+		expectedError                 bool
+		expectedRules                 []securityGroupRule
+		privateAPIWhitelistingEnabled bool
+		privateAPIWhitelistSubnets    string
```

Comment:
Is this var used? has sense to create a test case for private too?

Reply:
Test will be there with actual usage in code. For now just wiring things